indra/newview/llfloaterregioninfo.cpp

```diff
@@ -1394,7 +1395,7 @@ LLPanelRegionTerrainInfo::LLPanelRegionTerrainInfo()
     }
     for (S32 i = 0; i < LLTerrainMaterials::ASSET_COUNT; ++i)
     {
-        mLastSetMaterials[i] = LLUUID::null;
+        mLastSetMaterials[i] = LLGLTFMaterialList::BLANK_MATERIAL_ASSET_ID;
```

Comment:
We might want to move this value into indra_constants

Reply:
Good point, as that might prevent some race conditions in the future.